front/src/components/Step1/Step1.tsx

```diff
@@ -1,68 +1,71 @@
 import {Button} from '../ui/Button'
 import {useNavigate} from 'react-router-dom'
 import {Source, SourceSelector} from "../SourceSelector/SourceSelector.tsx";
-import {useState} from "react";
+import {useState,FC} from "react";
 import './Step1.scss'
 
 const sourceSelectors:Source[] = [
-    "instagram",
-    "x",
-    "medium",
-    "techCrunch",
-    "theGuardian" ,
-    "europeanParliament",
-    "nyTimes",
-    "googleDailyMix",
+  "instagram",
+  "x",
+  "medium",
+  "techCrunch",
+  "theGuardian",
+  "europeanParliament",
+  "nyTimes",
+  "googleDailyMix",
 ];
 
-export const Step1 = () => {
-    const navigator = useNavigate();
+export const Step1: FC = () => {
+  const navigator = useNavigate();
 
-    const [sources, setSources] = useState<Array<{ type: Source, param?: string }>>([]);
-    const [counter, setCounter] = useState(0);
+  const [sources, setSources] = useState<
+    Array<{ type: Source; param?: string }>
```

Comment:
change param to user input. Don't use shortened words which doesn't make sence. If we have other similar issues, fix them as well please